src/core/collector/candidate_record.rs

```diff
@@ -1,4 +1,4 @@
-// Copyright 2022 Parity Technologies (UK) Ltd.
+// Copyright 2023 Parity Technologies (UK) Ltd.
```

Comment:
```suggestion
// Copyright 2022-2023 Parity Technologies (UK) Ltd.
```

Reply:
In this case, it should be done globally in a separate commit/PR I suppose.